blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+---
+title: How to use Security Advisor with Skydive
+layout: blog-post
+author: Kalman Meth
+date: 31/07/2019
+---
+
+
+The Security Advisor filters the flow data obtained from Skydive, performs a data transformation, and saves the information to an object store in compressed JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+
+Instructions to developers to deploy and run the Secruity Advisor can be found at https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor.
+We present here extended instructions for users to deploy Skydive and the Security Advisor.
+
+We use the term "pipeline" to describe the sequence of operations performed on the data: classify, filter, encode, store.
+In the future, we plan to have additional pipelines performing different kinds of operations.
+
+## Deploy Skydive
+
+You can either download a statically linked version of Skydive or you can build it on your own.
+
+To download a pre-compiled statically linked version of Skydive, run the following command.
+
+```
+curl -Lo skydive https://github.com/skydive-project/skydive-binaries/raw/jenkins-builds/skydive-latest && chmod +x skydive && sudo mv skydive /usr/local/skydive/
+```
+
+To build your own version of Skydive, follow the instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the Skydive code.
+Enter the Skydive root directory and build the code.
+
+```
+cd $GOPATH/src/github.com/skydive-project/skydive
+make
+cp etc/skydive.yml.default /etc/skydive/skydive.yml
+// adjust settings in skydive.yml, if desired
+```
+
+The skydive binary is created in $GOPATH/bin/skydive.
+
+### All-in-one version
+
+```
+sudo skydive allineone -c /etc/skydive/skydive.yml
+```
+
+### Multi-node Deployment
+
+Alternatively, run a Skydive analyzer on one host and a Skydive agent on each of your hosts.
+
+On one machine:
+
+```
+sudo skydive analyzer -c /etc/skydive/skydive.yml
+```
+On each machine:
+
+```
+sudo skydive agent -c /etc/skydive/skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydive.yml to point to the analyzer.
+
+
+## Create Security Advisor (Pipeline) Configuration File
+
+Start from the secadvisor.yml.default in the secadvisor directory.
+
+
+```
+cp secadvisor.yml.default /etc/skydive/secadvisor.yml
+```
+
+The secadvisor.yml is used as a parameter when running secadvisor from the command line.
+
+The default secadvisor.yml has the following fields:
```

Comment:
including the entire content of the file here is counter productive as the format of the file is going to change rapidly. Rater just state and address parameters which typically require tweaking

Reply:
For user's who are not familiar, we need to describe the usage of each field. This is the only place (in the meantime) where each field is described.

Reply:
ok

Reply:
`secadvisor.yml`